api/desecapi/views.py

```diff
@@ -15,8 +15,7 @@
 from rest_framework.renderers import StaticHTMLRenderer
 from dns import resolver
 from django.template.loader import get_template
-from desecapi.authentication import (
-    BasicTokenAuthentication, URLParamAuthentication)
+from desecapi.authentication import *
```

Comment:
[explicit is better than implicit](https://www.python.org/dev/peps/pep-0020/) xD